src/info-label.scss

```diff
@@ -0,0 +1,98 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+/*!
+.fd-badge+(( --numeric | --icon ), ( --cozy | --compact ))
+*/
+$block: #{$fd-namespace}-info-label;
+
+.#{$block} {
+
+  // info-label DEFAULT
+  $fd-info-label-border-radius: 1.125rem !default;
+  $fd-info-label-color: var(--sapTextColor);
+  $fd-info-label-fontsize: var(--sapFontSmallSize);
+  $fd-info-label-default-padding: 0 0.625rem !default;
+  $fd-info-label-default-vertical-align: middle !default;
+  $fd-info-label-default-border-width: 1px !default;
+  $fd-info-label-default-border-style: solid !default;
+  $fd-info-label-default-height: 1.125rem !default;
+  $fd-info-label-default-text-transform: uppercase !default;
+  $fd-info-label-default-text-align: center !default;
+  $fd-info-label-default-letter-spacing: 0.0125rem !default;
+  $fd-info-label-default-display: inline-block !default;
+  $fd-info-label-default-text-padding-right: 0.3125rem !default;
+
+  // INFOLABEL ICON
+  $fd-info-label-icon-size: 0.625rem !default;
+  $fd-info-label-icon-padding-left: 0.3125rem !default;
+  $fd-info-label-icon-padding-right: 0.1875rem !default;
+  $fd-info-label-icon-padding: 0 0.5rem !default;
+
+  // INFOLABEL NUMERIC PADDING
+  $fd-info-label-numeric-padding: 0 0.3125rem !default;
+
+  // COZY AND COMPACT MARGIN
+  $fd-info-label-margin-compact: 0.4375rem 0 !default;
+  $fd-info-lable-margin-cozy: 0.8125rem 0 !default;
+
+  $color-accents: (
+    "1": ("background": #fdf3e6, "border": var(--sapAccentColor1)),
+    "2": ("background": #f9e6e6, "border": var(--sapAccentColor2)),
+    "3": ("background": #fce9f2, "border": var(--sapAccentColor3)),
+    "4": ("background": #f9ebf5, "border": var(--sapAccentColor4)),
+    "5": ("background": #eaeafa, "border": var(--sapAccentColor5)),
+    "6": ("background": #ebf3fa, "border": var(--sapAccentColor6)),
+    "7": ("background": #e8fafd, "border": var(--sapAccentColor7)),
+    "8": ("background": #f8fde7, "border": var(--sapAccentColor8)),
+    "9": ("background": #f2ebf9, "border": var(--sapAccentColor9)),
+    "10": ("background": #f1f3f4, "border": var(--sapAccentColor10))
+  );
+
```

Comment:
Keep in mind that the PR won't be merged until we have variables for the background colors from the sap-theming. They should be released soon.